Sources/MapboxCoreNavigation/NavigationRouteOptions.swift

```diff
@@ -17,11 +17,13 @@ open class NavigationRouteOptions: RouteOptions, OptimizedForNavigation {
 
      - seealso: `RouteOptions`
      */
-    public required init(waypoints: [Waypoint], profileIdentifier: ProfileIdentifier? = .automobileAvoidingTraffic) {
+    public required init(waypoints: [Waypoint], profileIdentifier: ProfileIdentifier? = .automobileAvoidingTraffic, queryItems: [URLQueryItem]? = nil) {
```

Comment:
I guess we don’t need to mention this in the changelog, since it’s just an override of the RouteOptions initializer in MapboxDirections, right?

Reply:
Yes, I think so